services/powchain/src/main/java/tech/pegasys/teku/services/powchain/PowchainConfiguration.java

```diff
@@ -41,11 +44,11 @@ public static Builder builder() {
   }
 
   public boolean isEnabled() {
-    return eth1Endpoint != null;
+    return eth1Endpoints != null && !eth1Endpoints.isEmpty();
```

Comment:
nit: should be able to omit the `!= null` check now since it should never be null (if it was the `validate` method would fail).
```suggestion
    return !eth1Endpoints.isEmpty();
```